code/modules/networks/computer3/computer3.dm

```diff
@@ -8,7 +8,7 @@
 	density = 1
 	anchored = 1.0
 	var/base_icon_state = "computer_generic"
-	var/temp = "<b>Thinktronic BIOS V2.1</b><br>"
+	var/temp = "Thinktronic BIOS V2.1\n"
```

Comment:
why no bold?

Reply:
i was not able to get the tags to work, so i took the tags off of the parts that cleared the screen and ran it through a proc that removed any other html tags